controllers/TweetController.js

```diff
@@ -30,29 +30,56 @@ res.status(200).json({getTweets , totalPages:Math.ceil(count / limit), currentPa
 
 
 
-// update
+// Update
 const updateTweets = async (req, res) => { 
     const newTweet = await Tweets.findById(req.params.id);
-    if (newTweet?.postBy === req.body.postBy) {
+    if(!newTweet){
+      return res.status(422).json('Tweet Not Available')
+    }
+    if (newTweet?.postBy._id.toString() === req.user._id.toString()) {
```

Comment:
Why `postBy._id.toString()`, `postBy` is supposed to just be the id of the user who added the tweet, are you storing the whole user object?

Reply:
Just id was not validating so i have to change it to string.